packages/components/modal/src/modal.tsx

```diff
@@ -155,7 +163,7 @@ export const Modal = motionForwardRef<ModalProps, "section">(
       size,
       ...props,
     })
-    const {
+    let {
```

Comment:
Please define the variable as const, similar to how it's done with`packages/components/modal/src/drawer-content.tsx` .